tests/plotting/test_nx_graph_export.py

```diff
@@ -0,0 +1,229 @@
+from typing import Any, cast
+
+import pandas as pd
+import pytest
+from rdkit import Chem
+from rdkit.Chem import AllChem
+
+from prolif.plotting.network.lignetwork import LigNetwork
+
+
+@pytest.fixture
+def simple_ligand_mol() -> Chem.rdchem.Mol:
+    """Create a simple ligand molecule (benzene) for testing"""
+    mol = Chem.MolFromSmiles("c1ccccc1")
+    mol = Chem.AddHs(mol)
+    AllChem.EmbedMolecule(mol, randomSeed=42)  # type: ignore
```

Comment:
```suggestion
    EmbedMolecule(mol, randomSeed=42)
```
Also please avoid using a plain `# type: ignore` in the future, nothing wrong with ignoring types but please specify the exact issue to ignore e.g. `# type: ignore[attr-defined]`